cmd/facette/js/graph.js

```diff
@@ -397,17 +397,26 @@ function graphDraw(graph, postpone, delay, preview) {
                     break;
                 }
 
+                var seenUnits = {};
```

Comment:
We tend to declare variable at the beginning of functions in our JS code.

It would be nice to put it with the other ones :smile: